operator/internal/manifests/internal/config/loki-config.yaml

```diff
@@ -1,4 +1,5 @@
 {{- /*gotype: github.com/grafana/loki/operator/internal/manifests/internal/config.Options*/ -}}
+{{ $shippersDic := shippers .ObjectStorage.Schemas }}
```

Comment:
I might be missing the point here, but constructing the map just to be able to check which shipper is used further down the template seems a bit overcomplicated to me. I'd see a function on the struct returning the shipper name ("boltdb" / "tsdb") or even just a boolean "UseTSDB" as an alternative with less overhead.

Reply:
As you suggested, I moved the logic into `manifests.ConfigOptions()`. I've also changed it to be a `[]string` but I had to use two booleans to avoid duplicates. Initially, I was using a map to avoid duplicates.